src/components/Issue.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import glamorous, { Div, Small } from 'glamorous'
+import { Link } from 'react-router-dom'
+
+import { spacing, fontSizes, fontWeights, lineHeights, colors } from '../theme'
+import {
+  IssueIcon,
+  PullRequestIcon,
+  MergeIcon,
+  CheckIcon,
+  XIcon,
+  CircleIcon,
+} from './Icon'
+
+const IssueTitle = glamorous(Link)({
+  fontSize: fontSizes[1],
+  fontWeight: fontWeights.bold,
+  lineHeight: lineHeights.tight,
+  color: colors.gray[8],
+  textDecoration: 'none',
+})
+
+function renderStateIcon(issue) {
+  if (issue.merged) {
+    return <MergeIcon color={colors.violet[8]} size={20} />
+  }
+
+  if (issue.mergeable) {
+    return (
+      <PullRequestIcon
+        color={issue.closed ? colors.red[8] : colors.green[8]}
+        size={20}
+      />
+    )
+  }
+
+  return (
+    <IssueIcon
+      color={issue.closed ? colors.red[8] : colors.green[8]}
+      size={20}
+    />
+  )
+}
+
+function renderStatusIcon(issue) {
+  const buildStatus =
+    issue.commits &&
+    issue.commits.commits[0].commit &&
+    issue.commits.commits[0].commit.status &&
+    issue.commits.commits[0].commit.status.state
+
+  if (buildStatus === 'SUCCESS') {
+    return <CheckIcon color={colors.green[8]} size={16} />
+  }
+
+  if (buildStatus === 'PENDING') {
+    return <CircleIcon color={colors.yellow[8]} size={12} strokeWidth={3} />
```

Comment:
Maybe we should have add a `SmallCircleIcon` so all these icons can have consistent size?